### DISCUSSION TITLE
Github Copilot plugin for Jetbrains Pycharm causing 'Waiting for process to detach' to hang on exit

### DISCUSSION THREAD

**Discussion by cercos** [community/community#88599] [Copilot Conversations]
When closing IDE and there are processes running (npm run watch or similar) it will ask to terminate.  When the plugin is enabled and uou click terminate, the 'waiting for process to detach' popup hangs.  I see the process terminate but it just hangs.

**Comment by montella1507** [comment]
https://youtrack.jetbrains.com/issue/IDEA-343537 

i can confirm. Check the issue above. Copilot is blocking ALL jetbrains IDEs
